docs/docs/PositionPaper.ThinkLike.RTOS.document.md

```diff
@@ -3,7 +3,7 @@ author: "Chris Gray"
 status: ReviewLevel2
 ---
 
-# Dynamic Telemetry and RTOS : COMING SOON
+# Applying RTOS techniques, to mitigate the risks of Dynamic Telemetry
```

Comment:
One way to think about this, dynamic telemetry is like a large scale, distributed database - all the queries have resource and time bounds, which are governed by privileges, access control and prioritization.

Reply:
+1 wanna write this section :)